releases/v20.2.0-alpha.2.md

```diff
@@ -0,0 +1,350 @@
+- Links returned in error messages to point to unimplemented issues now use the CockroachLabs redirect/short-link server. [#49836][#49836]
+
+### Enterprise edition changes
+
+- The default flush interval for [changefeeds](../v20.2/change-data-capture.html) that do not specify a `RESOLVED` option is now 5s instead of 200ms to more gracefully handle higher-latency sinks. [#49770][#49770]
+- [Full cluster restore](../v20.2/restore.html#full-cluster) is now more resilient to transient transaction retry errors while it restores. [#49950][#49950]
+- [`RESTORE`](../v20.2/restore.html) now remaps user-defined types within the backup to existing types in the cluster. [#50349][#50349]
+- `BACKUP` can now run in `detached` mode, which means the Job ID is now returned immediately without waiting for the `BACKUP` job to finish. [#50775][#50775]
+
+### SQL language changes
+
+- For Postgres compatibility, CockroachDB now supports the `ANALYZE <tablename>` statement that collects statistics on the given table for use by the [optimizer](../v20.2/cost-based-optimizer.html). The statement is functionally equivalent to the existing [`CREATE STATISTICS`](../v20.2/create-statistics.html) statement. [#49816][#49816]
+- Implemented the `ST_Transform`, `ST_Intersection`, `ST_PointOnSurface`, and `ST_Union`, `ST_Segmentize({geometry, float8})`, `ST_Project({geography,float8,float8})`, `ST_LongestLine({geometry, geometry})`, `ST_Azimuth({geometry,geometry})`, `ST_ShortestLine({geometry, geometry})`, `ST_Centroid ({geometry,bool})` functions. [#49783][#49783], [#49833][#49833], [#49827][#49827], [#49949][#49949], [#50018][#50018], [#50188][#50188], [#50292][#50292], [#50484][#50484]
+- Implemented the following builtins: `GeometryType`, `PostGIS_AddBBox`, `PostGIS_DropBBox`,  `PostGIS_Extensions_Upgrade`, `PostGIS_Full_Version`, `PostGIS_GEOS_Version`, `PostGIS_HasBBox`, `PostGIS_LibXML_Version`, `PostGIS_Lib_Build_Date`, `PostGIS_Lib_Version`, `PostGIS_Liblwgeom_Version`, `PostGIS_PROJ_Version`, `PostGIS_Scripts_Build_Date`, `PostGIS_Scripts_Installed`, `PostGIS_Scripts_Released`, `PostGIS_Version`, `PostGIS_Wagyu_Version`. [#50589][#50589]
+- Added support for casting `Geography` and `Geometry` types to and from `Bytes`/`JSONB`. [#50577][#50577]
+- Implemented `ST_AsGeoJSON` with options to show `bbox` and `CRS` information. [#49888][#49888]
+- Introduced `maxDecimalDigits` arguments for `ST_AsText` and `ST_AsEWKT`, which allow rounding of the decimal digits output in the `WKT` representation. [#49875][#49875]
+- Populated the `spatial_ref_sys` table with support SRID entries for geospatial data types. [#49887][#49887]
+- Implemented the `ST_Azimuth` functionality for the geography operator. [#50708][#50708]
+- `Geometry` now coerce invalid geography coordinates into correct geometry. [#50457][#50457]
+- Added support for `AddGeometryColumn` and other functions that mutate schema in `IMPORT PGDUMP`. [#50850][#50850]
```

Comment:
Rows 1 to here could be broken out into their own section, something like

> #### Geospatial changes
> ....
> #### General changes

I don't think it's strictly necessary (and would be a bit of work!) but could be helpful to folks following along with geospatial things